zipkin/pom.xml

```diff
@@ -28,7 +28,8 @@
 
   <properties>
     <main.basedir>${project.basedir}/..</main.basedir>
-    <main.java.version>1.6</main.java.version>
+    <!-- IntelliJ isn't clever enough to have different language levels for tests vs. main code -->
+    <main.java.version>1.8</main.java.version>
```

Comment:
obviously I will revert this. we can't willy nilly change zipkin's core library to java 8! It will break a few libraries